### PR TITLE
Fixed the plpy indentation bug

### PR DESCRIPTION
**Pull request by zifeishan**
Fixed a bug: when using 4-space indent in plpy extractors, the iterations are messed up. #104 

Added some tests for the fix.

Fixed a non-termination bug when executing SQL queries by a file and there are errors in the middle, which caused #73 .

### CODE REVIEW

File: src/test/python/test_ddext/input-1.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env python
+#! /usr/bin/env python
```
Comment (netj) [inline src/test/python/test_ddext/input-1.py]: Why is this line repeated?

File: src/test/python/test_ddext/input-2.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env python
+#! /usr/bin/env python
```
Comment (netj) [inline src/test/python/test_ddext/input-2.py]: Why is this line repeated as well?

**Comment by netj** [comment]
Looks good overall.  Adding tests for plpy certainly makes it easier to understand how ddext works, although I think we should avoid rewriting user's code and eventually redesign the plpy extractor interface to be more compatible with tsv extractors.

Btw how are the new tests triggered from the test.sh?  Can you wire that up and handle the inline comments as well?

**Comment by zifeishan** [comment]
@netj Fixed the duplicated lines.

The tests are now wired up with the main `test.sh` from DeepDive.

**Pull request merged**